src/sultan/api.py

```diff
@@ -92,25 +90,16 @@ def load(cls, cwd=None, sudo=False, user=None, hostname=None, env=None, logging=
             context['user'] = 'root' if sudo else getpass.getuser()
         context.update(kwargs)
 
-        s = Sultan(context=context)
-        return s
+        return cls(context=context)
```

Comment:
I don't think what you've done is wrong. It just feels more logical to use `cls`, the class object parameter, for instantiation.